src/@ionic-native/plugins/appodeal/index.ts

```diff
@@ -22,9 +22,9 @@ import { Injectable } from '@angular/core';
  */
 @Plugin({
   pluginName: 'Appodeal',
-  plugin: 'https://github.com/appodeal/appodeal-cordova-plugin',
+  plugin: 'https://github.com/appodeal/appodeal-cordova-plugin.git',
```

Comment:
Any specific reason  for adding the `.git` prefix here (and other occurrences below) ?